packit_service/worker/reporting.py

```diff
@@ -42,6 +42,9 @@
     "and be one of the first to know about all the news!",
     "Interested in the Packit team plans and priorities? "
     "Check [our epic board](https://github.com/orgs/packit/projects/7/views/29).",
+    "Do you use `propose_downstream`? We would be happy if you could help"
```

Comment:
I was going to mention pull-from-upstream and then I realized the users of it don't have Packit on upstream #fail